.github/pr-commands.json

```diff
@@ -0,0 +1,191 @@
+[
+  {
+    "type": "changedfiles",
+    "matches": {
+      "any": [
+        "/docs/",
+        "/contribute/"
+      ]
+    },
+    "action": "updateLabel",
+    "addLabel": "type/docs"
+  },
+  {
+    "type": "changedfiles",
+    "matches": {
+      "any": [
+        "/public",
+        "/packages",
+        "/e2e",
+        "/plugins-bundled",
+        "/scripts/build/release-packages.sh",
+        "/scripts/circle-release-next-packages.sh",
+        "/scripts/ci-frontend-metrics.sh",
+        "/scripts/grunt",
+        "/scripts/webpack",
+        "package.json",
+        "tsconfig.json",
+        "lerna.json",
+        ".babelrc",
+        ".prettierrc.js",
+        ".eslintrc",
+        "*.mdx"
+      ]
+    },
+    "action": "updateLabel",
+    "addLabel": "area/frontend"
+  },
+  {
+    "type": "changedfiles",
+    "matches": {
+      "any": [
+        "*.go",
+        "go.mod",
+        "go.sum",
+        "/contribute/style-guides/backend.md",
+        "/contribute/architecture/backend"
```

Comment:
Just an FYI - there's still a few others not in here yet such as`contribute/architecture/services.md` and `contribute/architecture/database.md` for example. But I think this is fine and we can fix those up and leave this as is

Reply:
https://github.com/grafana/grafana/pull/32041

Reply:
Nice thanks. I'll fix